config/routes.rb

```diff
@@ -74,6 +71,11 @@
   match '/manage/phone' => 'users/phones#update', via: [:patch, :put]
   get '/manage/recovery_code' => 'users/recovery_codes#show'
 
+  get '/otp/send' => 'users/two_factor_authentication#send_code'
+  get '/phone_setup' => 'users/two_factor_authentication_setup#index'
+  patch '/phone_setup' => 'users/two_factor_authentication_setup#set'
+  get '/users/two_factor_authentication' => 'users/two_factor_authentication#show', as: :user_two_factor_authentication
```

Comment:
I had to explicitly set a name here because we are using this method from an external lib https://github.com/Houdini/two_factor_authentication/blob/master/lib/two_factor_authentication/controllers/helpers.rb#L22

which expects the route to be named `user_two_factor_authentcation`